lib/howl/buffer.moon

```diff
@@ -277,6 +278,19 @@ class Buffer extends PropertyObject
       b_end_pos = @byte_offset end_pos
       @_buffer\ensure_styled_to pos: b_end_pos
 
+  mode_at: (pos) =>
```

Comment:
Well, my search over the code base may have missed some uses of `buffer.mode`...could you point out where exactly you're referring to?